articles/virtual-machines/linux/multiple-nics.md

```diff
@@ -176,4 +177,75 @@ You can also use a `copyIndex()` to then append a number to a resource name, whi
 You can read a complete example of [creating multiple NICs using Resource Manager templates](../../virtual-network/virtual-network-deploy-multinic-arm-template.md).
 
 ## Next steps
-Review [Linux VM sizes](sizes.md) when trying to creating a VM with multiple NICs. Pay attention to the maximum number of NICs each VM size supports. 
\ No newline at end of file
+Review [Linux VM sizes](sizes.md) when trying to creating a VM with multiple NICs. Pay attention to the maximum number of NICs each VM size supports. 
+
+## Additional Steps After A Linux VM with two NICs Got Created
+**Overview**
+
+When creating multiple NICs for a Linux Guest-OS based VM
+it is required to create additional routing rules which allows to send and receive traffic
+belonging to a specific NIC only. Otherwise traffic belonging to eth1 can not be processed correct, 
+due to the defined default route.  
+
+
+**Solution**
+
+At first it is required to add two new routing tables to the file /etc/iproute2/rt_tables
+
+    echo "200 eth0-rt" >> /etc/iproute2/rt_tables
+    echo "201 eth1-rt" >> /etc/iproute2/rt_tables
+
+To make the change persistent and get them applied during the network stack activation it is required 
+to alter the */etc/sysconfig/network-scipts/ifcfg-eth0* and */etc/sysconfig/network-scipts/ifcfg-eth1* file.
+Alter the line *"NM_CONTROLLED=yes"* to *"NM_CONTROLLED=no"*.
+Without this step the additonal rules/routing we are going to add taking no effect.
+In case it is is prefered to use NetworkManger dispatcher scripts please read 'man NetworkManager' for more details

```

Comment:
Leave out his line, as it introduces ambiguity. If the user wants to do that, they would know to go look for it and configure accordingly. A user that does not know about NetworkManager may now question it and wonder if they should use it.